tests/sandbox/linalg/test_linalg.py

```diff
@@ -152,3 +154,96 @@ def test_matrix_inverse_solve():
     node = matrix_inverse(A).dot(b).owner
     [out] = inv_as_solve.transform(None, node)
     assert isinstance(out.owner.op, Solve)
+
+
+def test_cholesky_dot_lower():
+    cholesky_lower = Cholesky(lower=True)
+    cholesky_upper = Cholesky(lower=False)
+
+    L = matrix("L")
+    L.tag.lower_triangular = True
```

Comment:
Can you parametrize the test so that one time we include the tag and the other not. When we don't include it we want to test the rewrite fails